specification/2.0/README.md

```diff
@@ -2310,12 +2288,10 @@ Additional properties are allowed.
 
 Application-specific data.
 
-> **Implementation Note:** Although extras may have any type, it is common for applications to
-store and access custom data as key/value pairs. As best practice, extras should be an Object
-rather than a primitive value for best portability.
```

Comment:
This `extras` implementation note is not in the schema, and is lost due to automatic regeneration.

Reply:
This was added in https://github.com/CesiumGS/wetzel/pull/42.